tests/testthat/test-csv.R

```diff
@@ -162,13 +162,13 @@ test_that("read_cmdstan_csv() returns correct diagonal of inverse mass matrix",
   skip_on_cran()
   csv_files <- c(test_path("resources", "csv", "model1-2-no-warmup.csv"))
   csv_output <- read_cmdstan_csv(csv_files)
-  expect_equal(as.vector(csv_output$inv_metric[[2]]),
+  expect_equal(as.vector(csv_output$inv_metric[[as.character(2)]]),
```

Comment:
In this example of only reading in the 2nd chain, the length of `csv_output$inv_metric` is now 1 (but the 1 element has the name "2") instead of length 2 with the first element NULL. So we're technically breaking backwards compatibility here. @rok-cesnovar Since we're still in beta (and this isn't the typical use case) I think it's ok.